multiplatform-markdown-renderer-m2/src/commonMain/kotlin/com/mikepenz/markdown/m2/Markdown.kt

```diff
@@ -0,0 +1,33 @@
+package com.mikepenz.markdown.m2
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.mikepenz.markdown.compose.components.MarkdownComponents
```

Comment:
Is MarkdownComponents missing from this PR?

Reply:
Shouldn't be 🤔. it's the interface from the core module.

Reply:
Ahhh, already existing. Nice.